SimulationAgent/Exceptions/DeviceActorNotInitializedException.cs

```diff
@@ -7,7 +7,7 @@ namespace Microsoft.Azure.IoTSolutions.DeviceSimulation.SimulationAgent.Exceptio
     public class DeviceActorNotInitializedException : Exception
```

Comment:
could you check if this exception is still used?

Reply:
I removed this, and another exception, that didn't have any references.